### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader: The action performed of copying is not announced by screen reader.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10643]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#NonBenchmark;#DesktopApp;#MAS4.1.3;#SQL Azure Data Studio;#FTP;#STP;#NVDA;#JAWS;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS4.1.3
Screen reader: NVDA2020.1, JAWS

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section on Welcome screen
4 .Launch screen reader as NVDA
5 .Navigate to documentation link hit enter and tab to copy button and press enter.

**Actual:**
On activating the copy button in the dialog the action performed is not announced by the screen reader.

**Issue observed in the following screens:**
**Videos Pop Up**

**Expected:**
When the copy button is activated screen reader should announce data has been copied and a visual cue as well should be provided.

**User Impact:**
Keyboard and screen reader users will not be aware of the important change that has occurred 

**Attachment link for Reference:**
[10643A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4705180/10643A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
@fsteffi I have confirmed such scenarios with Ryan Shugart before, if a button already states what it does and it finishes immediately, we don't have to announce that the copy is done.
closing as won't fix.

**Comment by fsteffi** [comment]
#closed;

on click on documentation now the screen is redirected to https://learn.microsoft.com/en-us/sql/azure-data-studio/?view=sql-server-ver16

Version: 1.43.0-insider (user setup)
Commit: b4d5a08199e9e0cb0138ce540c086ac32d072019
Date: 2023-03-07T06:31:08.002Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300
